sky/templates/oci-ray.yml.j2

```diff
@@ -0,0 +1,137 @@
+cluster_name: {{cluster_name}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.skylet.providers.oci.OCINodeProvider
+  region: {{region}}
+  cache_stopped_nodes: True
+
+auth:
+  ssh_user: ubuntu
+  ssh_private_key: {{ssh_private_key}}
+
+available_node_types:
+  ray_head_default:
+    resources: {}
+    node_config:
+      InstanceType: {{instance_type}}
+      VCPUs: {{cpus}}
+      MemoryInGbs: {{memory}}
+      AvailabilityDomain: {{zone}}
+      ImageId: {{image}}
+      AppCatalogListingId: {{app_catalog_listing_id}}
+      ResourceVersion: {{resource_version}}
+      Preemptible: {{use_spot}}
+      AuthorizedKey: {{ssh_public_key}}
+{% if num_nodes > 1 %}
+  ray_worker_default:
+    min_workers: {{num_nodes - 1}}
+    max_workers: {{num_nodes - 1}}
+    resources: {}
+    node_config:
+      InstanceType: {{instance_type}}
+      VCPUs: {{cpus}}
+      MemoryInGbs: {{memory}}
+      AvailabilityDomain: {{zone}}
+      ImageId: {{image}}
+      AppCatalogListingId: {{app_catalog_listing_id}}
+      ResourceVersion: {{resource_version}}
+      Preemptible: {{use_spot}}
+      AuthorizedKey: {{ssh_public_key}}
+{%- endif %}
+
+head_node_type: ray_head_default
+
+# Format: `REMOTE_PATH : LOCAL_PATH`
+file_mounts: {
+  "~/.oci/config": "~/.oci/config",
+  "~/.oci/oci_api_key.pem": "~/.oci/oci_api_key.pem",
+  "~/.sky/config.yaml": "~/.sky/config.yaml", 
```

Comment:
We will handle/upload this config file in our system. Please directly upload it within the cloud's logic.